src/views/app/detail-panel/preview/mail-preview.tsx

```diff
@@ -229,7 +160,66 @@ const MailContent: FC<{
 			disableTransition
 			data-testid="MailMessageRendererCollapse"
 		>
-			{message.isComplete && collapsedContent}
+			{message.isComplete && (
+				<Container
+					data-testid="MessageBody"
+					width="100%"
+					height="fit"
+					crossAlignment="stretch"
+					padding={
+						isInsideExtraWindow ? { vertical: 'small' } : { horizontal: 'large', vertical: 'small' }
+					}
+					background="gray6"
+				>
+					<Row>
+						<AttachmentsBlock
+							message={message}
+							isExternalMessage={isExternalMessage}
+							openEmlPreview={openEmlPreview}
+						/>
+					</Row>
+					<Padding style={{ width: '100%' }} vertical="medium">
+						{showAppointmentInvite ? (
+							<Container width="100%">
+								<InviteResponse
+									// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+									// @ts-ignore
+									onLoadChange={(): null => null}
```

Comment:
Currently the callback is totally useless and, on Calendars side, the callback is optional. Also defining it as an inline function will generate a new function on every render, causing a re-render of the inner component.
Since this hotfix is meant to resolve a re-render issue it makes sense to get rid of this old useless code

```suggestion
								
```